packages/cli/src/util/projects/get-project-by-id-or-name.ts

```diff
@@ -9,7 +9,7 @@ export default async function getProjectByNameOrId(
 ) {
   try {
     const project = await client.fetch<Project>(
-      `/projects/${encodeURIComponent(projectNameOrId)}`,
+      `/v8/projects/${encodeURIComponent(projectNameOrId)}`,
```

Comment:
This was necessary to avoid collisions with v1 routes. In the tests added here specifically, the path `/projects/list` would hit a specific route that listed all projects instead of retrieving a project named "list".